syz-manager/manager.go

```diff
@@ -576,19 +577,34 @@ func (mgr *Manager) isSuppressed(crash *Crash) bool {
 	return false
 }
 
+func (mgr *Manager) emailCrash(crash *Crash) {
+	if mgr.cfg.Email_addr != "" {
+		Logf(0, "sending email to %v", mgr.cfg.Email_addr)
+		rpt := fmt.Sprintf("%s", crash.Report.Report)
+		cmdstr := "mailx -s " + "\"Syzkaller: " + crash.Title + "\" " + mgr.cfg.Email_addr + " <<< \"" + rpt + "\""
+		cmd := exec.Command("bash", "-c", cmdstr)
+		err := cmd.Run()
+		if err != nil {
+			Logf(0, "Failed to send email: %v", err.Error())
```

Comment:
Log messages from non-capital letter please.
It's not necessary to do `err.Error()`, just passing `err` to `%v` will do the right thing.